### DISCUSSION TITLE
PaddleOCR/latest/applications/%E5%85%89%E5%8A%9F%E7%8E%87%E8%AE%A1%E6%95%B0%E7%A0%81%E7%AE%A1%E5%AD%97%E7%AC%A6%E8%AF%86%E5%88%AB

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14119] [Q&A]
# PaddleOCR/latest/applications/%E5%85%89%E5%8A%9F%E7%8E%87%E8%AE%A1%E6%95%B0%E7%A0%81%E7%AE%A1%E5%AD%97%E7%AC%A6%E8%AF%86%E5%88%AB

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/latest/applications/%E5%85%89%E5%8A%9F%E7%8E%87%E8%AE%A1%E6%95%B0%E7%A0%81%E7%AE%A1%E5%AD%97%E7%AC%A6%E8%AF%86%E5%88%AB.html?h=%E6%95%B0%E7%A0%81

**Comment by Cupcc** [comment]
下载百度网盘的v3_digital_model 无法正常导出模型。
```shell
[2024/10/30 10:26:19] ppocr INFO: load pretrain successful from ./pretrain_models/v3_digital_mobile/best_accuracy
Traceback (most recent call last):
  File "/to/PaddleOCR/tools/export_model.py", line 37, in <module>
    main()
  File "/to/PaddleOCR/tools/export_model.py", line 33, in main
    export(config)
  File "/to/PaddleOCR/ppocr/utils/export_model.py", line 374, in export
    export_single_model(
  File "/to/PaddleOCR/ppocr/utils/export_model.py", line 140, in export_single_model
    paddle.static.InputSpec(shape=[None] + input_shape, dtype="float32"),
                                  ~~~~~~~^~~~~~~~~~~~~
TypeError: can only concatenate list (not "NoneType") to list
```

**Reply by Cupcc** [reply]
查找了相关的issue,找到了解决方案采用 release 2.6版本的export_model.py 进行导出。 
你们能解释一下原因和新版的修复方案吗？

**Comment by UserWangZz** [comment]
因为范例模型比较早了，版本更新导致旧版模型无法使用新版本框架导出
